whenchanged/whenchanged.py

```diff
@@ -91,7 +92,7 @@ def run_command(self, thefile):
             print_message += '.' + now.strftime('%f') + ", running '" + ' '.join(self.command) + "'"
         if print_message:
             print('==> ' + print_message + ' <==')
-        subprocess.call(new_command, shell=(len(new_command) == 1))
+        subprocess.Popen(new_command, shell=(len(new_command) == 1), env=self.process_env)
```

Comment:
I think the behavior of `Popen` is slightly different from `call`, e.g. `Popen` doesn't wait for the command to complete. I think we should stick with `call` here. According to [the docs](https://docs.python.org/3/library/subprocess.html#subprocess.call) it accepts all the same arguments as `Popen`.

Reply:
Done.
Sorry I've missed this point in the doc.